crates/cairo-lang-sierra-to-casm/src/invocations/cheatcodes/call.rs

```diff
@@ -51,17 +52,19 @@ pub fn build_call(
         } into {
             return_data_start: return_data_start,
             return_data_end: return_data_end,
-            err_code: err_code
+            panic_data_start: panic_data_start,
+            panic_data_end: panic_data_end
         };
-        ap += 3;
-        jump Failure if err_code != 0;
+        tempvar failure = panic_data_end - panic_data_start;
+        ap += 4;
+        jump Failure if failure != 0;
```

Comment:
```suggestion
        ap += 4;
        jump Failure if panic_data_end != panic_data_start;
```
maybe there's no need for an additional `tempvar`?

Reply:
i actually tried but it doesn't work :<